remme/certificate/certificate_client.py

```diff
@@ -14,24 +14,32 @@
 # ------------------------------------------------------------------------
 
 import json
+import datetime
 
 from remme.protos.certificate_pb2 import CertificateStorage, \
     NewCertificatePayload, RevokeCertificatePayload, CertificateMethod
 from remme.shared.basic_client import BasicClient
 from remme.certificate.certificate_handler import CertificateHandler
 from remme.token.token_handler import TokenHandler
+from remme.certificate.certificate_handler import CERT_ORGANIZATION, CERT_MAX_VALIDITY
 
+from cryptography.x509.oid import NameOID
+from cryptography import x509
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.backends import default_backend
 
 class CertificateClient(BasicClient):
     def __init__(self):
         super().__init__(CertificateHandler)
 
     @classmethod
-    def get_new_certificate_payload(self, certificate_raw, signature_rem, signature_crt):
+    def get_new_certificate_payload(self, certificate_raw, signature_rem, signature_crt, cert_signer_public_key):
         payload = NewCertificatePayload()
         payload.certificate_raw = certificate_raw
         payload.signature_rem = signature_rem
         payload.signature_crt = signature_crt
+        if cert_signer_public_key:
```

Comment:
Ternary for sexiness, please :)

Reply:
No no no, we care about keeping our code readable, not "sexy". Ternaries in Python are quite hard to read and pretty uncommon.